packages/grpc-js/.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "root": true,
+  "extends": "./node_modules/gts",
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
I don't think this is right. `any` is used in several places, but it is used minimally and purposefully and I think it's valuable to need to add the comment to ignore the rule in every place where `any` is used.

Reply:
Currently, `any` is sometimes disabled for whole files (https://github.com/grpc/grpc-node/blob/e055468c299a1a22cc36b7760175b1964a3067d7/packages/grpc-js/src/load-balancing-config.ts#L26).
I think you want ignores for every line (would be 54 times)?

Reply:
No, what I have there is correct. The rule should be disabled for that whole file, but not for the whole package.

Reply:
Okay, but isn't disabling for the whole file also inconsequent? Is there any criteria, when to disable for the whole file, and when for the line? E.g. in [service-config.ts](https://github.com/grpc/grpc-node/blob/master/packages/grpc-js/src/service-config.ts) are 15 times `disable-next-line no-any`,  in contrast `load-balancing-config.ts` is only 3 times `any` used, but disabled for the whole file.

Reply:
In service-config.ts it's also disabled for the whole file, not per line. In both files, all of the functions take an `any` as an input and then validate every field to turn it into a typechecked output. In most other cases, `any` is appropriate to use in some functions in the file, but not in all of them.

Reply:
Done.